custom_model_runner/datarobot_drum/drum/utils.py

```diff
@@ -304,3 +304,11 @@ def capture_R_traceback_if_errors(r_handler, logger):
                 "t_traceback_err": traceback_exc,
             }
         raise
+
+
+def stringify_and_intify_label(label):
+    if isinstance(label, float):
+        if hash(label) != hash(int(label)):
+            raise ValueError("Class labels must be integers or strings")
```

Comment:
why _must_ labels be ints or strings? I know it's likely not common, but what's the issue with supporting labels like `1.1/2.2`?

Reply:
because I don't like them and they're bad

Reply:
I don't think we should restrict this. I could see some 'regression' datasets with a few unique target values that someone might want to try as multiclass. We support it in DR so we should support it in training models